sale_commission/__manifest__.py

```diff
@@ -2,8 +2,11 @@
 
 {
     'name': 'Sales commissions',
-    'version': '10.0.1.1.0',
-    'author': 'Odoo Community Association (OCA)',
+    'version': '10.0.2.0.0',
+    'author': 'Odoo Community Association (OCA),'
+              'Tecnativa,'
+              'AvanzOSC,'
+              'Agile BG',
```

Comment:
Thanks for the credit. Please use `Agile Business Group` if not too long.

Reply:
Yeah, you have worked a lot over these modules, so I think you deserve it.